media-libs/zita-convolver/zita-convolver-4.0.3.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+DESCRIPTION="C++ library implementing a real-time convolution matrix"
+HOMEPAGE="http://kokkinizita.linuxaudio.org/linuxaudio/"
+SRC_URI="http://kokkinizita.linuxaudio.org/linuxaudio/downloads/${P}.tar.bz2"
+
+LICENSE="GPL-3+"
+SLOT="0/4"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="sci-libs/fftw:3.0="
+DEPEND="${RDEPEND}"
+BDEPEND=""
+
+PATCHES=( "${FILESDIR}"/${PN}-4.0.3-makefile.patch )
+
+src_compile() {
+	emake -C source CXX="$(tc-getCXX)"
+}
+
+src_install() {
+	local myemakeargs=(
```

Comment:
configure in `src_configure`, install in `src_install`

Reply:
This is a Makefile-based build system. And all these argument only meaningful in `src_install` phase. So I don't see what's wrong here.